values.yaml

```
@@ -130,6 +130,13 @@ warehouse:
       # @ignore
       pullPolicy: IfNotPresent
 
+shell:
+  image:
+    registry: amrc-factoryplus.shef.ac.uk:5000
+    repository: factoryplus-shell
+    tag: 0.0.6
+    pullPolicy: IfNotPresent
```

@amrc-benmorrow how about this?

```suggestion
    registry: ghcr.io/amrc-factoryplus
    repository: acs-krb-utils
    tag: 1.0.0
    pullPolicy: IfNotPresent
```